### PR TITLE
Add issues closed line graph and update hovertemplate

### PR DESCRIPTION
**Pull request by christinaexyou**
@cdolfi Resolves part of https://github.com/oss-aspen/8Knot/issues/468 and https://github.com/oss-aspen/8Knot/issues/470 by making the following changes:

* Adds line for issues closed
* Updates the hover template with Date, indicating the time window for which prolificacy is calculated

### CODE REVIEW

**Review by cdolfi — CHANGES_REQUESTED** [review]
Lets add issue comments to this one too and rebase to main and will be good to go!

**Comment by christinaexyou** [comment]
@cdolfi Changes made:
* Added issue comment line
* Updated legend labels to be more concise - changed `PR Review Comment` to `PR Comment` and `PR Request Review` to `PR Review`

File: 8Knot/pages/contributors/visualizations/contrib_importance_over_time.py
Comment (cdolfi) [inline 8Knot/pages/contributors/visualizations/contrib_importance_over_time.py:21]: remove spaces

File: 8Knot/pages/contributors/visualizations/contrib_importance_over_time.py
Comment (cdolfi) [inline 8Knot/pages/contributors/visualizations/contrib_importance_over_time.py:70]: change back format

File: 8Knot/pages/contributors/visualizations/contrib_importance_over_time.py
Comment (cdolfi) [inline 8Knot/pages/contributors/visualizations/contrib_importance_over_time.py:84]: same on format

**Review by cdolfi — CHANGES_REQUESTED** [review]
Seems to be something weird happening format wise. Once that is fixed overall in the file I think it will be ready to merge (hard to see with current state, check out the "review" view if you want a good look)

**Comment by cdolfi** [comment]
@christinaexyou let me know when this is ready for re-review. If there is not an obvious way of fixing file wide this @JamesKunstle might have some guidance

**Comment by JamesKunstle** [comment]
@cdolfi @christinaexyou I'll pull this now and take a look to see if I have any ideas.

**Comment by JamesKunstle** [comment]
Working on this, though I'm getting a DBReadFailure for some reason, so I may need to spend some time debugging tomorrow before I can actually review this PR.

**Comment by JamesKunstle** [comment]
@christinaexyou 

## Review 

I pulled your most updated branch and tried the following orgs/repos, but neither 'Issues Closed' nor 'Issue Messages' lines appeared. 

'chaoss' org
'3scale' org
'kubernetes/kubernetes' repo

This is a screenshot of what I get for 'kubernetes/kubernetes':
Screenshot 2023-10-04 at 1 58 47 PM

And this is my current branch's log:
Screenshot 2023-10-04 at 2 01 02 PM

Which I believe is up to date.


## One additional request

I'd love it if the Action Type was in the hover information. That'd make reading the graph much easier.

**Comment by christinaexyou** [comment]
@cdolfi @JamesKunstle Changes made:
* Reverted formatting
* `Issue Closed` and `Issue Comment` line graphs are visible
* Shortened `PR Request ...` labels to `PR`
* Updated hover template to display action type

**Review by cdolfi — APPROVED** [review]
lgtm! Once james reviews we can merge

**Comment by JamesKunstle** [comment]
@christinaexyou Yuh this looks great, gonna merge it.

**Pull request merged**
